modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/ContentSearchTestCase.java

```diff
@@ -100,15 +101,16 @@
                             .encode(description, "UTF-8"));
             getPublisherAPIs.setHeader("Authorization", "Bearer " + accessToken);
             HttpResponse publisherResponse = client.execute(getPublisherAPIs);
-            if (getResultCount(publisherResponse) == 1) {
+            responseString = getResponseBody(publisherResponse);
+            if (getResultCount(responseString) == 1) {
                 Assert.assertTrue(true);
                 break;
             } else {
                 if (i == retries) {
-                    Assert.fail("Basic content search in publisher failed. Received response : " + publisherResponse);
+                    Assert.fail("Basic content search in publisher failed. Received response : " + responseString);
                 } else {
-                    log.warn("Basic content search in publisher failed. Received response : " + publisherResponse + " Retrying...");
-                    Thread.sleep(3000);
+                    log.warn("Basic content search in publisher failed. Received response : " + responseString + " Retrying...");
```

Comment:
Shall we check whether the length of this line exceeding our formatting rules?

Reply:
fixing in https://github.com/wso2/product-apim/pull/5009